### DISCUSSION TITLE
Plugins Timeline/Activity/Activity Mode

### DISCUSSION THREAD

**Discussion by AstreeDeshayes** [nasa/openmct#5207] [Community Q&A]
Hello,

I don't have the plugins "Timeline", "Activity" and "Activity Mode" on my OpenMCT version. Can someone tell me how to implement them please ?

Thanks,
Astrée

**Comment by shefalijoshi** [comment]
@AstreeDeshayes Have you pulled the latest version of Open MCT from npm? If that's the case, then you probably need to install the plugin to enable it.
For example: https://github.com/nasa/openmct/blob/master/index.html#L235

**Comment by akhenry** [comment]
@AstreeDeshayes  "Activity Mode" is a concept from the deprecated version of Open MCT timelines. Are you trying to update a legacy version of Open MCT?
